site/content/docs/manifest/pipeline.en.md

```diff
@@ -177,7 +180,29 @@ Ordered list of environments that your pipeline will deploy to.
 The name of an environment to deploy your services to.
 
 <span class="parent-field">stages.</span><a id="stages-approval" href="#stages-approval" class="field">`requires_approval`</a> <span class="type">Boolean</span>  
-Optional. Indicates whether to add a manual approval step before the deployment. Defaults to `false`.
+Optional. Indicates whether to add a manual approval step before the deployment (or the pre-deployment actions, if you have added any). Defaults to `false`.
+
+<span class="parent-field">stages.</span><a id="stages-predeployments" class="field">`pre_deployments`</a> <span class="type">Map</span>  
```

Comment:
could we add something like this to each of these fields from the env manifest page?
```
<span class="type">Map</span><span class="version">Added in [v1.30.0](../../blogs/release-v123.en.md#move-misplaced-http-fields-in-environment-manifest-backward-compatible)</span>
```